docker/entry.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+service postgresql start
+service postgresql status
+echo ">>>>>additional command: $@>>>>>"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)